meilisearch-release-changelog/v1.0.0.md

```diff
@@ -0,0 +1,121 @@
+# v1.0.0 release changelogs
+
+<!-- The following line should NOT be put in the official release changelogs -->
+⚠️ Since this is a release candidate (RC), we do NOT recommend using it in a production environment. Is something not working as expected? We welcome [bug reports](https://github.com/meilisearch/meilisearch/issues/new/choose) and [feedback about new features](https://github.com/meilisearch/product/discussions).
+
+We are proud to announce we are releasing the first stable release of Meilisearch! 🎉 This new release, contrary to v0.30.0, focus on stabilization instead of bringing a lot of new features. We took the opportunity to make the final touch regarding some breaking changes in the CLI but also in our error handler.
+Since Meilisearch is now stable, we will not introduce any new breaking changes in the future versions of Meilisearch until releasing v2.0.0. Here you have more information about what we consider [as breaking for Meilisearch](https://github.com/meilisearch/engine-team/blob/main/resources/versioning-policy.md).
+
+<!-- The following lines should ONLY be put in the official release changelogs -->
+🧰 Most integrations (SDKs, clients, libraries) will be compatible with this version within four hours of the release. Sometimes this can take up to 48 hours, depending on the issues we encounter during the release.
+
+<!-- The following lines should ONLY be put in the official release changelogs -->
+Here is the exhaustive list of integrations  **not immediately compatible** with v1.0.0:
+- TBD
+
+<!-- The following line should NOT be put in the official release changelogs -->
+📖 Meilisearch's official documentation will be ready for the official v1.0.0 release. Meanwhile, work-in-progress documentation is available [here]().
+
+# New features and improvements 🔥
+
+# Charabia
+
+TBD
+
+# Generate master key option
+
+You can now activate the `--generate-master-key` CLI option when launching Meilisearch.
+
+This option is particular since it will not launch Meilisearhc but will generate a safe-to-use master key for Meilisearch and will exit without any error.
```

Comment:
```suggestion
This option is particular since it won't launch Meilisearch but will generate a safe-to-use master key for Meilisearch and will exit without any error.
```